website/docs/installation.md

```diff
@@ -128,8 +128,8 @@ Please update to the latest Docusaurus 2 version shown at the top of the page, n
 
 ```json title="package.json"
 "dependencies": {
-  "@docusaurus/core": "^2.0.0-beta.0",
-  "@docusaurus/preset-classic": "^2.0.0-beta.0",
+  "@docusaurus/core": "^2.0.0-beta.2",
+  "@docusaurus/preset-classic": "^2.0.0-beta.2",
```

Comment:
```suggestion
  "@docusaurus/core": "^2.0.0-beta.0",
  "@docusaurus/preset-classic": "^2.0.0-beta.0",
```

It's not really sustainable to update those docs after each release, that's why we added a comment:

> Please update to the latest Docusaurus 2 version shown at the top of the page, not what is shown below.

If you want it to display the actual beta to use, we should rather use MDX instead and read from the `package.json` file to be sure it stays in sync over time

Reply:
Okay cool